src/commands/setup.rs

```diff
@@ -55,9 +53,9 @@ impl Setup {
 
         // Sysctl setup
         // set ip forwarding to 1 if not already
-        let sysctl_ipv4 = get_sysctl_value(IPV4_FORWARD)?;
+        let sysctl_ipv4 = core_utils::CoreUtils::lookup_sysctl_value(IPV4_FORWARD)?;
         if sysctl_ipv4 != *"1" {
-            set_sysctl_value(IPV4_FORWARD, "1")?;
+            core_utils::CoreUtils::apply_sysctl_value(IPV4_FORWARD, "1")?;
```

Comment:
Just wondering, what is the point of reading this value when we could just always set to 1.

Reply:
i was preventing the case where if it didnt exist, it would not blow up.  but maybe the api calls handle that and then in fact it would be better ... i'll double check that